contracts/Paper.sol

```diff
@@ -1,27 +1,34 @@
 pragma solidity ^0.8.4;
 
-contract Paper {
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract Paper is Ownable {
     enum State {onReview, Approved, Rejected}
```

Comment:
I think the state name could be more consistent. Maybe change `onReview` to `PendingReview` ?

```suggestion
    enum State {PendingReview, Approved, Rejected}
```